proto/cosmos/bank/v1beta1/tx.proto

```diff
@@ -101,6 +101,7 @@ message MsgUpdateParamsResponse {}
 // message are left unchanged.
 //
 // Since: cosmos-sdk 0.47
+// HV2: Not used in Heimdall since multiple denoms are not supported
```

Comment:
I wonder if deleting this message type and regenerating the protobuf types would make sense. It would reduce the code's footprint, but it would also introduce a bunch of deletions of unused code.

It depends on how important it is to keep the code as close to the vanilla SDK as possible.